newscoop/template_engine/metaclasses/MetaSubtitle.php

```diff
@@ -297,14 +297,22 @@ public static function ProcessImageLink(array $p_matches)
 
         try {
             $templatesService = \Zend_Registry::get('container')->getService('newscoop.templates.service');
-            $smarty = $templatesService->getSmarty();
-            $uri->uri_parameter = "image $imageOptions";
-            $smarty->assign('imageDetails', $detailsArray);
-            $smarty->assign('MediaRichTextCaptions', $preferencesService->MediaRichTextCaptions);
-            $smarty->assign('uri', $uri);
-            $smarty->assign('imgZoomLink', $imgZoomLink);
-
-            return $templatesService->fetchTemplate("editor_image.tpl");
+            $templatesService->setVector(array(
+                'publication' => $request->attributes->get('_newscoop_publication_metadata[alias][publication_id]', null, true), 
```

Comment:
The variable $request does not exist. https://scrutinizer-ci.com/g/takeit/Newscoop/inspections/a8920bff-10e9-400d-a88a-a46d049db872/issues/files/newscoop/template_engine/metaclasses/MetaSubtitle.php?status=new&orderField=path&order=asc